web.py

```diff
@@ -30,7 +31,13 @@
     module_path = 'ext.app.{}'.format(app_file)
     import_module(module_path)
 except Exception as e:
-    helpers.log(str(e))
+    helpers.log(''.join(traceback.format_exception(None, e, e.__traceback__)))
```

Comment:
I haven't run this locally yet, but I think you can just use `helpers.logger.exception(e)` here. See: https://docs.python.org/3/library/logging.html#logging.exception

`logger.exception()` should automatically print the stacktrace alongside the exception message.

Reply:
Ah, I thought there must be something better than what I found. I've switched that over now and tested it.